dictionaries/css/src/css.txt

```diff
@@ -1134,8 +1138,9 @@ y
 yellow
 yellowgreen
 z
-za
 zA
+za
 zero
+zindex
```

Comment:
it's hyphenated https://developer.mozilla.org/en-US/docs/Web/CSS/z-index

Reply:
That is correct normally z-index has a minus sign in the middle.
But developers use it without a dash in [CSS/SCSS] variables.

Reply:
that would be a typo ...

Reply:
Do you suggest I remove it from this PR and make PRs for the different projects that use zindex as a single word?

Reply:
I removed it as it could lead to confusion.